tests/phpunit/tests/test-settings-list-tables.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+class Settings_List_Tables_Test extends PLL_UnitTestCase {
+
+	/**
+	 * @param WP_UnitTest_Factory $factory
+	 */
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		parent::wpSetUpBeforeClass( $factory );
+
+		self::create_language( 'en_US' );
+		self::create_language( 'fr_FR' );
+	}
+
+	public function test_display_languages_table() {
+		wp_set_current_user( 1 );
+
+		$GLOBALS['hook_suffix'] = 'settings_page_mlang';
+		$_GET['page'] = 'mlang';
+
+		$links_model = self::$model->get_links_model();
+		$pll_env = new PLL_Settings( $links_model );
```

Comment:
I think it could be moved into `set_up()` using predefined prop `$this->pll_env`.

Reply:
Oh I see, maybe with a method like `init_pll_settings( $page_name )` then?

Reply:
Good idea.